### PR TITLE
doc: fix typo and add bold weight to first mention of important terms in r.geomorphon manual page

### PR DESCRIPTION
**Pull request by gregorywaynepower**
* Adds bolding around the first mention of search radius (L) and relief threshold (d) since the variables are referenced later on in the description.
* Fixes typo in sentence starting with "...There are 3**8 = 6561 possible ternary patterns..."
* Changes "However by eliminating all patterns that are results of either rotation or reflection of other patterns wa set of 498 patterns remain referred as geomorphons. This is a comprehensive and exhaustive set of idealized landforms that are independent of the size, relief, and orientation of the actual landform." to "However by eliminating all patterns that are results of either rotation or reflection of other patterns _**we get a**_ set of 498 patterns remain referred as geomorphons. This is a comprehensive and exhaustive set of idealized landforms that are independent of the size, relief, and orientation of the actual landform."

### CODE REVIEW

**Review by veroandreo — APPROVED** [review]
Thanks!

File: raster/r.geomorphon/r.geomorphon.html [hunk 1]
```diff
@@ -23,7 +23,7 @@ <h3>What is geomorphon:</h3>
 nadir angles along the eight principal compass directions. The ternary
 operator converts the information contained in all the pairs of zenith and
 nadir angles into the ternary pattern (8-tuple). The result depends on the
-values of two parameters: search radius (L) and relief threshold (d). The
+values of two parameters: <b>search radius (L)</b> and <b>relief threshold (d)</b>. The
```
[the remark this answers is not in the document]
Comment (veroandreo) [reply raster/r.geomorphon/r.geomorphon.html:26]: Is it possible that relief threshold in the text refers to the `flat` parameter?

[the remark this answers is not in the document]
Comment (gregorywaynepower) [reply raster/r.geomorphon/r.geomorphon.html:26]: I have no problems with reverting the bolding change.

**Review by neteler — APPROVED** [review]
Thanks!

**Comment by neteler** [comment]
I tried to backport it but get:

```bash
12:12 $ git push upstream releasebranch_8_4
Enumerating objects: 46, done.
Counting objects: 100% (31/31), done.
Delta compression using up to 12 threads
Compressing objects: 100% (13/13), done.
Writing objects: 100% (19/19), 4.29 KiB | 4.29 MiB/s, done.
Total 19 (delta 14), reused 9 (delta 6), pack-reused 0 (from 0)
remote: Resolving deltas: 100% (14/14), completed with 10 local objects.
remote: error: GH013: Repository rule violations found for refs/heads/releasebranch_8_4.
remote: Review all repository rules at https://github.com/OSGeo/grass/rules?ref=refs%2Fheads%2Freleasebranch_8_4
remote: 
remote: - This branch must not contain merge commits.
remote:   Found 1 violation:
remote: 
remote:   cf219d0854cf3c67b547295547f606b1506ab2bd
remote: 
To github.com:OSGeo/grass.git
 ! [remote rejected]       releasebranch_8_4 -> releasebranch_8_4 (push declined due to repository rule violations)
error: failed to push some refs to 'github.com:OSGeo/grass.git'
```

Does anyone know how to get rid of this problem?

**Comment by echoix** [comment]
> I tried to backport it but get:
> 
> 
> 
> ```bash
> 
> 12:12 $ git push upstream releasebranch_8_4
> 
> Enumerating objects: 46, done.
> 
> Counting objects: 100% (31/31), done.
> 
> Delta compression using up to 12 threads
> 
> Compressing objects: 100% (13/13), done.
> 
> Writing objects: 100% (19/19), 4.29 KiB | 4.29 MiB/s, done.
> 
> Total 19 (delta 14), reused 9 (delta 6), pack-reused 0 (from 0)
> 
> remote: Resolving deltas: 100% (14/14), completed with 10 local objects.
> 
> remote: error: GH013: Repository rule violations found for refs/heads/releasebranch_8_4.
> 
> remote: Review all repository rules at https://github.com/OSGeo/grass/rules?ref=refs%2Fheads%2Freleasebranch_8_4
> 
> remote: 
> 
> remote: - This branch must not contain merge commits.
> 
> remote:   Found 1 violation:
> 
> remote: 
> 
> remote:   cf219d0854cf3c67b547295547f606b1506ab2bd
> 
> remote: 
> 
> To github.com:OSGeo/grass.git
> 
>  ! [remote rejected]       releasebranch_8_4 -> releasebranch_8_4 (push declined due to repository rule violations)
> 
> error: failed to push some refs to 'github.com:OSGeo/grass.git'
> 
> ```
> 
> 
> 
> Does anyone know how to get rid of this problem?

When you updated your local branch, did you create a merge commit locally when pulling (and you did the commit before updating the local branch)

**Comment by neteler** [comment]
> When you updated your local branch, did you create a merge commit locally when pulling (and you did the commit before updating the local branch)

I see (almost): and how to get out of this?

**Comment by neteler** [comment]
Solved with local support by @mmacata :)

**Pull request merged**